### PR TITLE
Run tests *before* installing in tmproot

### PR DESCRIPTION
**Pull request by fcremo**
CMake components: execute tests before installing them in tmproot.

### CODE REVIEW

File: .orchestra/config/lib/cmake.lib.yml [hunk 1]
```diff
@@ -101,6 +100,8 @@
     fi
     (@ end -@)
 
+    cmake --install "$BUILD_DIR"
+
     (@- if post_install_script: @)
```
Comment (aleclearmind) [inline .orchestra/config/lib/cmake.lib.yml:105]: I think `@` is eating spaces.

```
[+] INFO - The following script is going to be executed:
cmake --build "$BUILD_DIR" ${JOBS:+-j$JOBS}cmake --install "$BUILD_DIR"
```

Which leads to:

```
[+] ERROR - Script failed with exit code 1
Output stream:
'-j' invalid number '8cmake' given.
Unknown argument --install
Unknown argument /builds/gitlab/revng/orchestra/orchestra/build/libfort/optimized
```

Comment (fcremo) [reply .orchestra/config/lib/cmake.lib.yml:105]: It's not the `@`, but the `-`, and yep, this one should be removed `(@- if test: @)`

Comment (fcremo) [reply .orchestra/config/lib/cmake.lib.yml:105]: Actually, also the one in `(@ end -@)`

Comment (aleclearmind) [reply .orchestra/config/lib/cmake.lib.yml:105]: Can you push a fix?

**Comment by fcremo** [comment]
@aleclearmind fixed `(@-` whitespace eating disorder

**Comment by aleclearmind** [comment]
Merged in 481c778. Thanks!

**Pull request closed without merging**